ingestion/src/metadata/ingestion/source/database/databricks/metadata.py

```diff
@@ -661,3 +663,41 @@ def get_table_description(
                 f"Table description error for table [{schema_name}.{table_name}]: {exc}"
             )
         return description
+
+    def _filter_owner_name(self, owner_name: str) -> str:
+        """remove unnecessary keyword from name"""
+        pattern = r"\(Unknown\)"
+        filtered_name = re.sub(pattern, "", owner_name).strip()
+        return filtered_name
+
+    def get_owner_ref(self, table_name: str) -> Optional[EntityReferenceList]:
+        """
+        Method to process the table owners
+        """
+        try:
+            query = DATABRICKS_GET_TABLE_COMMENTS.format(
+                schema_name=self.context.get().database_schema,
+                table_name=table_name,
+            )
+            result = self.connection.engine.execute(query)
+            owner = None
+            for row in result:
+                row_dict = dict(row)
+                if row_dict.get("col_name") == "Owner":
+                    owner = row_dict.get("data_type")
+                    break
+            if not owner:
+                return
+
+            owner = self._filter_owner_name(owner)
+            owner_ref = None
+            try:
+                owner_email = EmailStr._validate(owner)
+                owner_ref = self.metadata.get_reference_by_email(email=owner_email)
+            except Exception:
```

Comment:
1. we dont need to catch ANY exception here no? only the one triggered by wrong strings inside `EmailStr`
2. also, I don't think we need to run `._validate`. You might just be able to do `EmailStr(<string>)` and see if it blows up or not

Reply:
1. in case of different invalid emails it shows different error(Valueerror, SyntaxError) so I thought to apply general exception.
2. I checked with EmailStr(string) but it's not working in that way (maybe with v2 it's changed)
[change](https://github.com/pydantic/pydantic/issues/798#issuecomment-1992089526)